### DISCUSSION TITLE
Can we run UNO with winui app as background task and activate  after some time or background event.

### DISCUSSION THREAD

**Discussion by contact2vikas** [unoplatform/uno#8336] [Q&A]
Can we run UNO with winui app as background task and activate  after some time or background event.

**Comment by jeromelaban** [comment]
Can you specify on which platform you want background tasks to be used?

**Reply by contact2vikas** [reply]
I want to use my application in  Windows and Mac OS. I am creating app using Uno + win ui3

**Comment by pkar70** [comment]
Background work, i.e. Timer and some system triggers, are implemented for Android in PR#2655.

**Comment by contact2vikas** [comment]
I want to use my application in  Windows and Mac OS. I creating app using
Uno + win ui3
